main_test.go

```diff
@@ -27,14 +27,3 @@ func Test_hasAChickenWord(t *testing.T) {
 		t.Errorf("chicken is a chicken")
 	}
 }
-
-func Test_hasAGrittyWord(t *testing.T) {
-
-	if hasAGrittyWord(strset.New("moist")) {
-		t.Errorf("moist is not gritty")
```

Comment:
I think we still need to validate that `moist` isn't gritty, unless @DotNetGeekette can confirm grittiness.

Reply:
https://github.com/fitzoh/jamie-chicken/blob/c23269adbf936081951518ca840951187cabf8cf/sentiment_test.go#L47-L53